tests/kuttl/tests/dataplane-deploy-tls-test/00-dataplane-create.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands: |
+  oc deploy -f ./deploy/certs-create.yaml
```

Comment:
I'm afraid you have to use `script` subfield for this.
https://kuttl.dev/docs/testing/steps.html#shell-scripts